repos/fdbt-site/src/pages/ticketConfirmation.tsx

```diff
@@ -280,44 +329,28 @@ export const buildPeriodOrMultiOpTicketConfirmationElements = (
         }
     }
 
-    const addProduct = (product: Product | MultiProduct): void => {
-        const productDurationText = `${
-            product.productDurationUnits
-                ? `${product.productDuration} ${product.productDurationUnits}${
-                      product.productDuration === '1' ? '' : 's'
-                  }`
-                : `${product.productDuration}`
-        }`;
-
-        if (!product.productDuration) {
-            throw new Error('User has no product duration and/or validity information.');
-        }
-
-        const content = [
-            `Price - £${product.productPrice}`,
-            `Duration - ${productDurationText}`,
-            ...getCarnetDetailsContent(product.carnetDetails),
-        ];
-
-        confirmationElements.push({
-            name: `${product.productName}`,
-            content,
-            href: 'multipleProducts',
-        });
-    };
-
     const periodExpiryAttribute = getSessionAttribute(ctx.req, PERIOD_EXPIRY_ATTRIBUTE);
 
     if (!periodExpiryAttribute || !isPeriodExpiry(periodExpiryAttribute)) {
         throw new Error('Either periodExpiryAttribute is undefined or not type expected');
     }
 
-    if (isArray(products)) {
-        products.forEach(product => {
-            addProduct(product);
-        });
-    } else if (!isArray(products)) {
-        addProduct(products);
+    if (!pointToPointPeriod) {
+        const { products } = getSessionAttribute(ctx.req, MULTIPLE_PRODUCT_ATTRIBUTE) as MultipleProductAttribute;
```

Comment:
Do you need the type assertion here?

Reply:
@RobCatton

Reply:
We could throw an error if the products is the MultipleProductAttributeWithErrors type or undefined @Aaron-Collis-IW